src/OmniSharp.Abstractions/Models/v1/FilesChanged/FilesChangedRequest.cs

```diff
@@ -1,8 +1,13 @@
 using System.Collections.Generic;
+using Newtonsoft.Json;
 using OmniSharp.Mef;
 
 namespace OmniSharp.Models.FilesChanged
 {
-    [OmniSharpEndpoint(OmniSharpEndpoints.FilesChanged, typeof(IEnumerable<Request>), typeof(FilesChangedResponse))]
-    public class FilesChangedRequest : IRequest { }
+    [OmniSharpEndpoint(OmniSharpEndpoints.FilesChanged, typeof(IEnumerable<FilesChangedRequest>), typeof(FilesChangedResponse))]
+    public class FilesChangedRequest : Request
+    {
+        [JsonConverter(typeof(FileChangeTypeConverter))]
```

Comment:
Couldn't this just be [`StringEnumConverter`](https://github.com/JamesNK/Newtonsoft.Json/blob/f012ba857f36fe75b1294a210b9104130a4db4d5/Src/Newtonsoft.Json/Converters/StringEnumConverter.cs)?

Reply:
I think that's right.